src/LanguageMapper.js

```diff
@@ -0,0 +1,52 @@
+'use strict'
+
+const { ALL_LANGUAGES, UNDEFINED_LANGUAGE, BOKMAAL, NORWEGIAN } = require('./LanguageConstants')
+const convertString = string => string !== null ? string.trim().toLowerCase() : ''
+
+const mappedValue = language => {
+  if (language === null || language === undefined) {
+    return UNDEFINED_LANGUAGE
+  }
+  if (NORWEGIAN === language) {
+    return BOKMAAL
+  }
+  return language
+}
+
+const getLanguageByIso6391Code = code => mappedValue(
+  ALL_LANGUAGES.find(language => convertString(code).match(language.iso6391Code))
+)
+
+const getLanguageByIso6392Code = code => mappedValue(
+  ALL_LANGUAGES.find(language => language.iso6392Codes.find(variant => convertString(code).match(variant)))
+)
+
+const getLanguageByIso6393Code = code => mappedValue(
+  ALL_LANGUAGES.find(language => convertString(code).match(language.iso6393Code))
+)
+
+const getLanguageByEnglishName = name => mappedValue(
+  ALL_LANGUAGES.find(language => convertString(name).match(new RegExp(language.eng.toLowerCase())))
```

Comment:
Ingen praktisk betydning, men ser ut som dette er (minst) like enkelt. gjelder alle sjekker i denne fila
```suggestion
  ALL_LANGUAGES.find(language => convertString(name) === convertString(language))
```